state-chain/pallets/cf-swapping/src/tests/fill_or_kill.rs

```diff
@@ -61,7 +61,7 @@ fn both_fok_and_regular_swaps_succeed_first_try() {
 fn price_limit_is_respected_in_fok_swap() {
 	const SWAPS_ADDED_BLOCK: u64 = 1;
 	const SWAPS_SCHEDULED_FOR_BLOCK: u64 = 3;
-	const SWAP_RETRIED_AT_BLOCK: u64 = SWAPS_SCHEDULED_FOR_BLOCK + SWAP_RETRY_DELAY_BLOCKS as u64;
+	const SWAP_RETRIED_AT_BLOCK: u64 = SWAPS_SCHEDULED_FOR_BLOCK + DEFAULT_SWAP_RETRY_DELAY_BLOCKS;
```

Comment:
why are we using the default for these tests instead of the retry delay that's in storage?

Reply:
I did try that, but to get the value from storage it needs to be done within a `execute_with` context, but if we do that we cant use the value as an argument in the `then_execute_at_block`. So it becomes very difficult and messy. Best option was to just use a default constant, since the constant sets the mock genesis, it should not get out of sync unless the test directly changes the storage.